platform/viewer/src/components/ViewportGrid/ConnectedViewportGrid.js

```diff
@@ -2,15 +2,20 @@ import ViewportGrid from './ViewportGrid.js';
 import { MODULE_TYPES } from '@ohif/core';
 import { connect } from 'react-redux';
 import { extensionManager } from './../../App.js';
+import memoize from 'lodash/memoize';
 
-const mapStateToProps = state => {
+const getAvailableViewportModules = memoize(viewportModules => {
```

Comment:
Good point. If the viewportModules is the same value, so to would be the result of `getAvailableViewportModules`